poetry_to_uv.py

```diff
@@ -145,6 +154,12 @@ def modify_authors_line(dumped_txt: str) -> str:
     return new.replace("\\", "")
 
 
+def project_license(uv_toml: dict):
+    if l := uv_toml["project"].get("license"):
+        if isinstance(l, str):
+            uv_toml["project"]["license"] = {"text": l}
```

Comment:
The license is always a string, right? So either we don't test for it, or we need an else clause with a warning of sorts.

Reply:
I believe it is always a string, although I'm not entirely certain. Therefore, I was considering overwriting it only when it is confirmed as a string. There is certainly an opportunity to add warnings.